frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -257,9 +257,16 @@ const composeAndPushNotificationMessages = (
             getSelectedRole()?.role_name
         );
     }
-    if (errorIdList.length > 0) {
+
+    if(errorIdList.errorMsg) {
+        setNotificationMsg(
```

Comment:
Hi Jota, since we handle all the notification message in the "setGrantAccessNotificationMsg" method in the "NotificationState.ts" file, maybe let's pass this error message as a parameter, and still set the notification over there.

Reply:
Hello, I'll do that!